src/plugins/vis_type_timeseries/public/application/components/vis_editor_visualization.js

```diff
@@ -194,18 +182,13 @@ class VisEditorVisualizationUI extends Component {
           />
         </EuiFlexItem>
 
-        {panelInterval && (
-          <EuiFlexItem grow={false}>
-            <EuiText color="default" size="xs">
-              <p>
-                <FormattedMessage
-                  id="visTypeTimeseries.visEditorVisualization.panelInterval"
-                  defaultMessage="Interval: {panelInterval}"
-                  values={{ panelInterval }}
-                />
-              </p>
-            </EuiText>
-          </EuiFlexItem>
+        {this.isAnyPanelTypeExceptTimeseries() && (
+          <DataTimeRangeModeLabel
+            seriesData={seriesData}
+            panelInterval={panelInterval}
+            modelInterval={model.interval ?? AUTO_INTERVAL}
+            modelTimeRangeMode={model.time_range_mode}
+          />
```

Comment:
This label is getting added to the TSVB editor, but I was kind of hoping we could add it to the TSVB _visualization_ itself. This is definitely an improvement, but it won't help improve the understanding in a dashboard.

Reply:
@wylieconlon just to clarify this. You propose to display the indicator only for the Last Value and not for the Entire timerange that I def agree and I understand the need for having this indicator also for the embeddables. But maybe we have users that don't want this to be displayed on the dashboard. I am just making assumptions here but I would love to hear your thoughts.

Reply:
@wylieconlon It seems to me that it will not be quite correct to show any additional icons in the **Embeddable** mode. In this mode, I expect to see only data based on the configured visualization (agree, we are not showing values for overrided index pattern, panel filters, time field and so on).
Also, if we decide for some reasons to add it, then I would like to be able to hide that icon it in **Embeddable**  mode but it produce new configuration properties....
In addition, I don't understand how this icon should look, maybe the UX team can help us.

My suggestion is to move on with this RP + change the default from `Last value` -> `Entire timerange soon `

Reply:
There is already one design for embeddables that are using _custom_ time ranges, which is basically what "last value" is doing:

<img width="292" alt="Screen Shot 2021-03-02 at 6 46 25 PM" src="https://user-images.githubusercontent.com/666475/109730528-9e34e480-7b87-11eb-97ba-ad4c4dba489d.png">

Without this indicator on top of the embeddable, users would be confused when comparing the two numbers side by side. So I think this PR isn't doing enough unless we can add it to the embeddable in some way- but I agree that we should have this be something users can hide if they want to.